tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -11446,17 +11446,32 @@ struct FloatInputUintOutput
 };
 
 TEST_F(ExecutionTest, IsNormalTest) {
-    // EnableShaderBasedValidation();
-    // In order, the input is -Zero, Zero, -Denormal, Denormal, -Infinity, Infinity, -NaN, Nan, and then 4 Normal float numbers.
-    // Only the last 4 floats are normal, so we expect the first 8 results to be 0, and the last 4 to be 1, as defined by IsNormal.
-    std::vector<float> Validation_Input_Vec = {-0.0, 0.0, -(FLT_MIN / 2), FLT_MIN / 2, -(INFINITY), INFINITY, -(NAN), NAN, 530.99f, -530.99f, 122.101f, -.122101f};
-    std::vector<float> *Validation_Input = &Validation_Input_Vec;
+  // EnableShaderBasedValidation();
+  // In order, the input is -Zero, Zero, -Denormal, Denormal, -Infinity, Infinity, -NaN, Nan, and then 4 Normal float numbers.
+  // Only the last 4 floats are normal, so we expect the first 8 results to be 0, and the last 4 to be 1, as defined by IsNormal.
+    
+  D3D_SHADER_MODEL sm = D3D_SHADER_MODEL_6_0;
+  LogCommentFmt(L"\r\nVerifying isNormal in shader "
```

Comment:
It's not typical to announce this in the test log, is this needed?

Reply:
No, it's likely a remnant of dynamic indexing, I'll remove it.